### PR TITLE
release-2.8-Webctl

### PR DESCRIPTION
**Pull request by resmaa**
**2.8 Release Feature**

Web-based kubectl added as part of cluster management

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/856#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/3-cluster-management/12-palette-webctl.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Overview
+
+Palette leverages Kubectl through an in-built command line interface for the users to communicate with their workload clusters. This enables our users to deploy applications, inspect and manage cluster resources, and view logs using the Palette terminal without an external terminal. 
+
+# Usage Scenarios
+
+* Simple Cluster Access
```
Comment (shantanubansal) [inline content/docs/04-clusters/3-cluster-management/12-palette-webctl.md:22]: We can drop "Simple" keyword

File: content/docs/04-clusters/3-cluster-management/12-palette-webctl.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+3. Click the `Connect` button available at the `Kubernetes Config File.` 
+
+
+4. Wait for the terminal to be launched and start communicating to the cluster using the `kubectl` commands.
+
+
+## Cluster Access with OIDC Authentication Enabled
+
+The cluster access with OIDC authentication enables the clients to verify the end user's identity before establishing cluster connectivity. The user needs to establish an OIDC-based authentication to the cluster through an Identity provider of their choice. To establish cluster access with OIDC authentication, follow the steps below:
```
Comment (shantanubansal) [inline content/docs/04-clusters/3-cluster-management/12-palette-webctl.md:47]: We should also mention.. 
To make this work, they need to add **https://console.spectrocloud.com/v1/shelly/callback** in their IDP app as a callback/redirectURI

File: content/docs/04-clusters/3-cluster-management/12-palette-webctl.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Overview
+
+Palette leverages Kubectl through an in-built command line interface for the users to communicate with their workload clusters. This enables our users to deploy applications, inspect and manage cluster resources, and view logs using the Palette terminal without an external terminal. 
+
+# Usage Scenarios
+
+* Simple Cluster Access
```
Comment (shantanubansal) [inline content/docs/04-clusters/3-cluster-management/12-palette-webctl.md:22]: Can we remove this Simple Key Word?

**Review by shantanubansal — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/856#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/856#pullrequestreview-1096587194" title="Approved">shantanubansal</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**